hack/docker-compose.acceptance-tests.yml

```diff
@@ -1,5 +1,4 @@
 ---
-version: "2.1"
```

Comment:
What's the benefit of dropping this attribute?

Reply:
No benefit I guess, it's just that I noticed it's deprecated now:
* https://docs.docker.com/compose/compose-file/04-version-and-name/
* https://github.com/compose-spec/compose-spec/issues/13

Reply:
Makes sense.